sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -198,6 +198,11 @@ _current AS (
           THEN FALSE
         ELSE TRUE
       END AS reported_metrics_ping,
+      CASE
+        WHEN baseline.client_id IS NULL
+          THEN FALSE
+        ELSE TRUE
+      END AS reported_baseline_ping,
```

Comment:
Should we filter where this column is TRUE in the view, to return only baseline ping numbers?

Reply:
Can you elaborate on this thought?  Did the request include this change?

Reply:
The users should query the view. The question is if we want to prefilter in the view only baseline ping or users apply the filter when needed? @bani

Reply:
I'm waiting to hear from Ruoxi, but I like the filter.

Reply:
-1 on adding this to the view. We would expect to see clients who only reported e.g. first_session pings in this table (and we use it as such)

Reply:
@rhythmofrain63 any thoughts on this?

Reply:
+1 on pre-filter